### ISSUE TITLE
Design | Profile | Notification settings | Finalize MHV notification labels

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#59721]
## Background

We need to update [our mock-ups to reflect the latest copy and IA for MHV notifications](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-notification-preferences.md).

### The three notification types in question are:
- medical images + reports
- secure messaging
-  MHV Newsletter Notification

### Medical images + reports
- **Current name in MHV:** VA medical images and reports available notification
- **Proposed notification group for VA.gov:** Your health care
- **Proposed name for VA.gov:** Requested medical images and reports available to download
- **Trigger:** VA medical images and reports available notification is a message triggered only after a user has requested a specific set of images in the MHV “Download my data > Medical images and reports” section (details below). 

#### Details
There’s a table in that section that lists all the available image and radiology reports, and each one has up to three options: `View Report`, `View Images`, and `Request`. If the user clicks request, it triggers the process to get the images in a downloadable format (unsure of exactly what systems/APIs are called). On the front-end, the user sees a modal saying “Your request is currently processing. It may take some time to retrieve your VA Medical Images and Reports. In the meantime, you can choose to [update your profile](link to profile) to get an email notifying you when they are ready to view.”

This screenshot is the modal that appears after you click "Request Images" when looking at the table of all your images and reports. Note: Reports are immediately available when entered, but the user always has to "request" the images. I think "request" in this sense is more of a load/generate action than actually submitting a request for approval.
<img src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/6e58d240-905e-471e-a4eb-57b9073f7c14" width="50%" />

Here's the actual email that is sent:
>Dear [User],
The VA Medical Images and Report that you requested are now available. The information will be accessible for 3 days. After 3 days, you will need to resubmit your request.
To view your VA medical Images and Report in your browser:
log into My HealtheVet
navigate to the VA medical images and reports page
select a link in the I want to… column
Please do not reply directly to this email as it is an automated message from the My HealtheVet System.

<img src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/e256d833-10cb-4d91-b2e6-11fb11f60a19" width="50%" />

With the current flow, a notification name like "Requested medical images and reports available to download" sets better expectations about what the notification actually is.

### Secure messaging
- **Current name is MHV**:  "New Secure Message Notifications"
- **Proposed notification group for VA.gov**: Your health care
- **Proposed name for VA.gov**: New secure message available
- **Trigger:** Provider has sent new secure message

#### Details
Email text:
>Dear XYZ,
You have a new Secure Message waiting to be read. Sign into MyHealtheVet to read your message.

>From: *ZEGEREK_LMNOPQRSTUVWXYZ
Message ID: 123456789

>For online help:
[hours]
[phone number]

>This is an automated message. Please do not reply to this email.

<img src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/c2c7177c-c463-4211-81bf-41daf93f4f53" width="50%" />

### MHV newsletter notification
- **Current name in MHV**:  "Subscribe to Bi-Weekly Email Newsletter"
- **Proposed notification group for VA.gov**: General VA information and updates
- **Proposed name for VA.gov**: My HealtheVet bi-weekly newsletter
- **Proposed opt in label for VA.gov**: Subscribe by email
- **Trigger:** MHV team schedules a newsletter

## Tasks
- [x] Review current IA and content w/ CAIA
- [x] Update sketch files to reflect the correct name for the MHV secure message and medical images and reports notification items
- [x] Update sketch file to reflect the correct language for the MHV newsletter notification channel
- [x] Get any content changes reviewed/approved by stakeholders
- [x] Incorporate any changes to mock-ups (this may include going back to CAIA) 
- [x] Notify FE/PM when complete
- [ ] Make a separate ticket to update use case docs once it's launched

**Comment by andaleliz** [comment]
@adamwhitlock1 I'm looking at the Communication Permissions API and second guessing whether or not the notification channel labels come from the API or not. Do you know if the words "notify me by text" that are next to the radio buttons today come from the FE or via the service?

**Comment by andaleliz** [comment]
@mtcA6 all of the relevant sketch files are linked from [this doc](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-notification-preferences.md).

**Comment by adamwhitlock1** [comment]
Just to have it in writing, since we just communicated about it during our sprint planning:

The labels for each notification item "notify me by text" NO NOT come directly from the api and are something that we have explicitly put into the code of the UI for each notification.


An example of the way the api data is sent:

``` js
{
  "id": "",
  "type": "hashes",
  "attributes": {
    "communicationGroups": [
      {
        "id": 1,
        "name": "Board of Veterans' Appeals",
        "description": null,
        "communicationItems": [
          {
            "id": 6,
            "name": "Appeal status updates",
            "communicationChannels": [
              {
                "id": 1,
                "name": "Text",
                "description": "SMS Notification",
                "defaultSendIndicator": true,
                "communicationPermission": {
                  "id": 18994,
                  "allowed": false
                }
              }
            ]
          },
          {
            "id": 1,
            "name": "Appeal hearing reminders",
            "communicationChannels": [
              {
                "id": 1,
                "name": "Text",
                "description": "SMS Notification",
                "defaultSendIndicator": true,
                "communicationPermission": {
                  "id": 8132,
                  "allowed": true
                }
              }
            ]
          }
        ]
      },
      {
        "id": 4,
        "name": "Payments",
        "description": "Payments to the Veteran",
        "communicationItems": [
          {
            "id": 5,
            "name": "Disability and pension deposit notifications",
            "communicationChannels": [
              {
                "id": 1,
                "name": "Text",
                "description": "SMS Notification",
                "defaultSendIndicator": false
              }
            ]
          }
        ]
      },
      {
        "id": 3,
        "name": "Your health care",
        "description": null,
        "communicationItems": [
          {
            "id": 3,
            "name": "Appointment reminders",
            "communicationChannels": [
              {
                "id": 1,
                "name": "Text",
                "description": "SMS Notification",
                "defaultSendIndicator": true,
                "communicationPermission": {
                  "id": 18955,
                  "allowed": false
                }
              }
            ]
          },
          {
            "id": 4,
            "name": "Prescription shipment and tracking updates",
            "communicationChannels": [
              {
                "id": 1,
                "name": "Text",
                "description": "SMS Notification",
                "defaultSendIndicator": true,
                "communicationPermission": {
                  "id": 18993,
                  "allowed": false
                }
              }
            ]
          },
          {
            "id": 7,
            "name": "RX refill shipment notification",
            "communicationChannels": [
              {
                "id": 2,
                "name": "Email",
                "description": "Email Notification",
                "defaultSendIndicator": false
              }
            ]
          },
          {
            "id": 8,
            "name": "VA Appointment reminders",
            "communicationChannels": [
              {
                "id": 2,
                "name": "Email",
                "description": "Email Notification",
                "defaultSendIndicator": false
              }
            ]
          },
          {
            "id": 9,
            "name": "Securing messaging alert",
            "communicationChannels": [
              {
                "id": 2,
                "name": "Email",
                "description": "Email Notification",
                "defaultSendIndicator": false
              }
            ]
          },
          {
            "id": 10,
            "name": "Medical images and reports available",
            "communicationChannels": [
              {
                "id": 2,
                "name": "Email",
                "description": "Email Notification",
                "defaultSendIndicator": false
              }
            ]
          }
        ]
      },
      {
        "id": 2,
        "name": "General VA Updates and Information",
        "description": null,
        "communicationItems": [
          {
            "id": 11,
            "name": "Biweekly MHV newsletter",
            "communicationChannels": [
              {
                "id": 2,
                "name": "Email",
                "description": "Email Notification",
                "defaultSendIndicator": false
              }
            ]
          }
        ]
      },
      {
        "id": 5,
        "name": "QuickSubmit",
        "description": "QuickSubmit Notifications",
        "communicationItems": [
          {
            "id": 12,
            "name": "QuickSubmit Upload Status",
            "communicationChannels": [
              {
                "id": 1,
                "name": "Text",
                "description": "SMS Notification",
                "defaultSendIndicator": true
              }
            ]
          }
        ]
      }
    ]
  }
}
```

**Comment by andaleliz** [comment]
Inquired about MHV notifications in [this slack thread](https://dsva.slack.com/archives/C04DRS3L9NV/p1686334008074209) to learn more about medical images + reports, and secure messaging. Will reach out to Carnetta if I don't get any answers

**Comment by andaleliz** [comment]
Learned more about the MHV secure message notification:
1. The current label in the MHV UI is "New Secure Message Notifications"
2. The contents of the email are:

Dear XYZ,
You have a new Secure Message waiting to be read. Sign into MyHealtheVet to read your message. 

From: *ZEGEREK_LMNOPQRSTUVWXYZ
Message ID: 123456789

For online help:
[hours]
[phone number]

This is an automated message. Please do not reply to this email.

**Comment by mtcA6** [comment]
on the preference, when we add it to profile, will we need some version of this copy warning users of the response lag time? 
![Image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/6d667c06-2a98-4f00-94a1-4f20d894c6f2)

**Comment by andaleliz** [comment]
My gut reaction is that I don't think so, since the notification setting an alert when a new message is received. It feels more appropriate to set that expectation when someone is sending a message. But I can be convinced otherwise!

**Comment by mtcA6** [comment]
@andaleliz I could be convinced either way as well :) 
I agree, it's more appropriate when you're sending a message. 

In terms of surfacing relevant links, it might be good to link users to the secure messaging center from the Secure Message preference in the profile, so they can set the preference and then easily navigate to secure messaging

**Comment by andaleliz** [comment]
I learned more about the MHV "VA Medical Images and Reports" notification.  

The “VA medical images and reports available notification” is a message triggered only after a user has requested a specific set of images in the MHV “Download my data > Medical images and reports” section. There’s a table that lists all the available image and radiology reports, and each one has up to three options:  View Report, View Images, and Request.  If they click request, it triggers the process to get the images in a downloadable format (unsure of exactly what systems/APIs are called) and modal saying “Your request is currently processing. It may take some time to retrieve your VA Medical Images and Reports. In the meantime, you can choose to [update your profile](link to profile) to get an email notifying you when they are ready to view.”

IMO this whole flow feels a little clunky and I think could be refined, but that's out of scope for our team.  With the current flow, a notification name like "Requested medical images and reports available to download" sets better expectations about what the notification actually is.

**Comment by mtcA6** [comment]
@andaleliz i know you asked around and I swear somewhere I saw a message about the newsletter and the copy for that. Can you help me locate that? I also remember us talking about this in one of the collab cycle calls as well.

**Comment by andaleliz** [comment]
@mtcA6 I asked around about the first two bullets, but I don't think I asked about the third. It did come up in the collab cycle as something that needs to be figured out, but we haven't worked on that yet.

**Comment by andaleliz** [comment]
Created CAIA ticket #62900 to start the process to get content support (we do not need to track this ticket on our board)

**Comment by andaleliz** [comment]
Status update: I reviewed with the CAIA folks yesterday, and we made these decisions:
- **Medical images + reports notification item name:** Requested medical images available to download
- **Secure messaging notification item name::** New secure message available
- **MHV Newsletter:** 
  - this should go into the **General VA information and updates** notification group (which is currently hidden on prod and probably staging) since it is applicable to people who aren't enrolled in VA healthcare. 
  - The  **General VA information and updates** notification group should be the last group on the page, since it is the least important to Veterans
  - **notification item name:** My HealtheVet bi-weekly newsletter
  - **channel label**: "Subscribe by email"

**Comment by andaleliz** [comment]
CAIA had some more thoughts and we've made the following updates:

- Changed name of notification group from Your Health Care to Health Care
- Updated name of imaging notification to "Notifications when medical images you requested are available"
- Order of notifications will be:
  - Health appointment reminders
  - New secure message available
  - Prescription shipping and tracking updates
  - Notifications when medical images you requested are available

The Sketch files have all been updated. 

@mtcA6 I forgot one of the tasks for this ticket is to get review from MHV stakeholders, so this will carry over into the next sprint. Can you help facilitate that review? I'm not sure exactly who all we need to consult, (my guess is the folks listed in the project outline, and maybe Coulton Bunny) or if it needs to be a meeting or can be done async. 

I estimate 2 more points carrying over for next sprint.

**Comment by mtcA6** [comment]
@andaleliz trying to decide the best approach. 

I'm happy to help and I'm torn if we should do something asynchronous, wait for the Wednesday touchpoint call next week w MHV and VA Profile, or some other combination of options.

**Comment by Samara-Strauss** [comment]
Bringing the exact copy over from #62900 so I can find it more easily:

### Your health care

- Health appointment reminders
- New secure message available
- Prescription shipping and tracking updates
- Requested medical images available to download

### General VA information and updates 
- My HealtheVet bi-weekly newsletter

**Comment by andaleliz** [comment]
@Samara-Strauss there were changes captured in the last comment on #62900, and in this ticket that weren't reflected in your comment above.  I edited your comment so it shows the latest.

**Comment by andaleliz** [comment]
@mtcA6 In my experience with the MHV folks, their calendars are packed, as are their inboxes 🙃  Any OCTO reviewers would probably be fine to work with either way.

Perhaps a starting point could be to send an email with mock-ups, and ask for review/approval that way. if that proves to be ineffective, then you could try to schedule a call.

The scheduled touchpoint does seem like a good opportunity but I'm not sure if:
1. any OCTO MHV to VA.gov folks need to approve this
2. those people are on those calls

Another risk is that Carnetta or some other MHV person can't make that meeting - the few that I've been in, Carnetta is not there.

**Comment by Samara-Strauss** [comment]
@andaleliz we sit in on a biweekly MHV and VA Profile sync, so we can bring this up then. Is there a mockup with the updated copy? I know we still have an issue with the appointment reminders and prescriptions notifications needing two different entries for text and email, so just let me know what you have mocked up right now.

**Comment by andaleliz** [comment]
@Samara-Strauss yes, all of the mock-ups have the updated copy. I think [this](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/a/vRLvAZ1) would be the best one to show them.  This represents all the notification options a user would see if they have all their contact info on file, receive health care, and are associated with a text-notification VAMC.
 
I didn't know appointment reminders were in the same bucket as Rx shipments and need a separate entry. Is there a link you can share for me to learn more about that?

**Comment by Samara-Strauss** [comment]
This is perfect! Thanks, Liz!

> I didn't know appointment reminders were in the same bucket as Rx shipments and need a separate entry. Is there a link you can share for me to learn more about that?

No link, but I am pretty sure appointment reminders are powered by a different source for VEText than the MHV email reminders, but since I am having trouble finding documentation, I will need to confirm this.

**Comment by andaleliz** [comment]
I'm going to move this to the blocked column since we won't be having this review until 8/30 (the next sync meeting)

**Comment by andaleliz** [comment]
Carnetta approved the updated notification names today! I have updated all the Sketch files, and updated the comment above with all the notifications names that Samara was referencing to have it all in one place.

Over and out.

**Comment by mtcA6** [comment]
@andaleliz could you link the corresponding sketch files that were updated? 

Although, for posterity's sake, since we do modify our sketch files, should we upload some sort of static pdf that shows the end decisions that were here?

**Comment by andaleliz** [comment]
Sure, here's a [desktop example](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/a/AZnL7Wx) and here's [mobile](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/a/vRLvAZ1).  But, all of the artboards in the sketch file have been updated (there's a lot).

In my experience static references of design mock-ups end up being more confusing/churn-causing, than they are helpful. I'd really rather not.

The comment above can serve as a source of truth for the work on this ticket, which was to finalize the content.

**Comment by mtcA6** [comment]
@andaleliz ok, thanks. I've created a [use case issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65272) for the open task. 

Wednesday we can discuss with MHV and VA Profile on the bi-weekly. You've got Carnetta's signoff, now we need to get this into the VA Profile queue so even if MHV folks aren't able to join again at least we can get this moving w/VA Profile.

cc: @Samara-Strauss
